Tevinthuku/Politico
# Ft setup party viewership page 163549189

**What does this PR do?**
This PR adds a page where any user can view all political parties present in the  platform

**Describe tasks to be completed**
1. Render a styled list of political parties and the date of their creation

**How can this be manually tested?**
1. Clone this repository
2. Visit `party-viewership.html` page on any browser and you will see the rendered list of political parties

**PT stories relevant to this PR?**

[#163549189](https://www.pivotaltracker.com/n/projects/2241721/stories/163549189)


![image](https://user-images.githubusercontent.com/12128153/52080629-4e028700-25a9-11e9-8537-7b97a3849cc4.png)
